### PR TITLE
Add participant id to catalog

### LINKED ISSUE #196: Introduce `participantId` on `Catalog` level
For #195 we need a `participantId` on the `Catalog` level. This gives context to dataset->has_policy and will be used e.g. in `ContractRequestMessage->dspace:offer->assigner`

--
Matthias Binzer

### PR DESCRIPTION
**Pull request by jimmarino**
Adds "dspace:participantId" to the DCAT Catalog response.

Closes #196

### CODE REVIEW

**Review by sebbader-sap — CHANGES_REQUESTED** [review]
There are also adjustments needed in https://github.com/International-Data-Spaces-Association/ids-specification/blob/main/catalog/message/schema/catalog-schema.json and https://github.com/International-Data-Spaces-Association/ids-specification/blob/main/catalog/message/shape/dcat-shapes.ttl. If you make it a branch of this repo, I can push the fixes directly.

File: catalog/catalog.protocol.md [hunk 1]
```diff
@@ -77,6 +77,10 @@ The following table lists well-know endpoint types:
 
 Note that the property `dcat:servesDataset` should be omitted from the `DataService` since [Datasets](../model/terminology.md#dataset) are included as top-level entries. Clients are not required to process the contents of `dcat:servesDataset`.
 
+#### 1.1.4 Participant Id
+
+The identifier of the participant providing the `Catalog` is specified using the `dspace:participantId` attribute on that `DCAT Catalog`.
```
Comment (juliapampus) [inline catalog/catalog.protocol.md:82]:
```suggestion
The identifier of the participant providing the [Catalog](../model/terminology.md#catalog) is specified using the `dspace:participantId` attribute on that [DCAT Catalog](https://www.w3.org/TR/vocab-dcat-3/#Class:Catalog).
```
Consistent usage of links.

File: catalog/message/example/catalog.json
Comment (juliapampus) [inline catalog/message/example/catalog.json:1]: As this JSON is changed.. shouldn't this also result in changes to the schemas and shapes?

**Review by juliapampus — CHANGES_REQUESTED** [review]

File: catalog/message/example/catalog.json
Comment (jimmarino) [reply catalog/message/example/catalog.json:1]: Yes @sebbader-sap will be fixing those

**Review by sebbader-sap — APPROVED** [review]

**Review by juliapampus — APPROVED** [review]

**Pull request merged**